### DISCUSSION TITLE
A Request to Add MiThoriumSSI support

### DISCUSSION THREAD

**Discussion by neogen6** [ROM-builders/temporary#23942] [General]
my local manifest for lineage-20.0: https://github.com/neogen6/local_manifests/blob/lineage-20.0/lineage-20.0.xml

upstream local manifest: https://github.com/Mi-Thorium/local_manifests

example build: https://cirrus-ci.com/build/5542712833736704

Closest thing I could find is this comment: https://github.com/ROM-builders/temporary/discussions/9810#discussioncomment-2784248

**Comment by Apon77** [comment]
Use 1 space at lunch line after lunch command, your script has 2 spaces.

**Reply by neogen6** [reply]
It worked, thanks!
